### DISCUSSION TITLE
[v5] Mail-Provider. Multiple receivers

### DISCUSSION THREAD

**Discussion by rbartholomay** [adonisjs/core#3062] [Help] [answered]
Hi there,

is it possible to send mails to multiple receivers? It seems that the docu explain only one receiver and when is try a comma separated string of receivers it will failed.

Any idea?

**Comment by thetutlage** [comment, marked as answer]
Just call the `to` method for multiple times

**Comment by rbartholomay** [comment]
Thatˋs the way I currently do. I thought that I am missing something…

thanks!
